spec/hyrax/acl/access_control_list_spec.rb

```diff
@@ -38,4 +63,139 @@
       end
     end
   end
+
+  describe 'grant DSL' do
+    let(:mode) { :read }
+    let(:user) { Hyrax::Acl::Agent.new('user1') }
+    let(:group) { Hyrax::Acl::Agent.new("group/public") }
```

Comment:
There is a `Hyrax::Acl::Group` class now; might be better to use that in this test.